src/Common/MetricInterface.php

```diff
@@ -17,4 +17,11 @@ public function getName(): string;
      * @return float
      */
     public function resolve(): float;
+
+    /**
+     * Returns metric tags.
+     *
+     * @return string[]
+     */
+    public function getTags(): array;
```

Comment:
should not be just string array, because tags are map `tagname=value`

Reply:
`string[]` adiitonally allows numeric keys and I see this worth for simplicity of signatures. imventing some kind of tag bag would make us to create in evey case we want to use tags